ansible/roles/kong-api/defaults/main.yml

```diff
@@ -5913,6 +5914,27 @@ kong_apis:
     config.required: false
     config.enabled: false
 
+- name: assessmentItemRead
+  uris: "{{ assessment_prefix }}/v1/items/read"
+  upstream_url: "{{ learning_service_url }}/assessment/v3/items/read"
```

Comment:
Shouldn't this be `assessment_service_url`?

Reply:
@keshavprasadms  Ideally it should be `assessment_service_url` but we are using old assessment APIs so we kept as `learning_service_url`. I think this should be fine for old assessment APIs

@AmiableAnil Please correct me if i'm wrong.

cc: @vinukumar-vs

Reply:
Assessment item read API is in learning service that's why it is learning_service_url. It's still not migrated to assessment-service.

Reply:
Still its not correct url. https://github.com/project-sunbird/sunbird-devops/blob/c52efdae72e7e337d030921ba34cf3dcccf513ad/ansible/roles/kong-api/defaults/main.yml#L101
You should use https://github.com/project-sunbird/sunbird-devops/blob/c52efdae72e7e337d030921ba34cf3dcccf513ad/ansible/roles/kong-api/defaults/main.yml#L102

Reply:
@keshavprasadms Updated the URL.